tasks/section_1/cis_1.1.x.yml

```diff
@@ -1,51 +1,84 @@
 ---
 
-- name: "1.1.2 | PATCH | Ensure /tmp is configured"
-  systemd:
-      name: tmp.mount
-      daemon_reload: true
-      enabled: true
-      masked: false
-      state: started
+- name: "1.1.2 | AUDIT | Ensure /tmp is configured"
+  block:
+      - name: "1.1.2 | AUDIT | Ensure /tmp is configured"
+        ansible.builtin.shell: findmnt -n /tmp
+        changed_when: false
+        failed_when: false
+        register: amazon2_tmp_exists
+
+      - name: "1.1.2 | WARN | Ensure /tmp is configured | Absent"
+        ansible.builtin.debug:
+            msg: "Warning!! /tmp is configured to use fstab but does not exist"
+        changed_when: true
+        when: amazon2_tmp_exists.stdout | length == 0
   when:
-      - amazon2cis_tmp_svc
       - amazon2cis_rule_1_1_2
+      - not amazon2cis_tmp_svc
   tags:
-      - level1
+      - level1-server
+      - level1-workstation
       - automated
-      - patch
-      - rule_1.1.2
+      - audit
       - mounts
+      - rule_1.1.2
 
 - name: |
-         "1.1.3 | PATCH | Ensure noexec option set on /tmp partition
-          1.1.4 | PATCH | Ensure nodev option set on /tmp partition
-          1.1.5 | PATCH | Ensure nosuid option set on /tmp partition"
-  mount:
+        "1.1.3 | PATCH | Ensure noexec option set on /tmp partition"
+        "1.1.4 | PATCH | Ensure nodev option set on /tmp partition"
+        "1.1.5 | PATCH | Ensure nosuid option set on /tmp partition"
+  ansible.posix.mount:
       name: /tmp
       src: "{{ item.device }}"
       fstype: "{{ item.fstype }}"
       state: present
-      opts: defaults,{% if amazon2cis_rule_1_1_4 %}nodev,{% endif %}{% if amazon2cis_rule_1_1_3 %}noexec,{% endif %}{% if amazon2cis_rule_1_1_5 %}nosuid{% endif %}
-  notify: systemd restart tmp.mount
-  with_items:
-      - "{{ ansible_mounts }}"
+      opts: defaults,{% if amazon2cis_rule_1_1_3 %}noexec,{% endif %}{% if amazon2cis_rule_1_1_4 %}nodev,{% endif %}{% if amazon2cis_rule_1_1_5 %}nosuid{% endif %}
+  notify: Remount_tmp
```

Comment:
Remount_tmp is not in handlers but the old systems restart tmp.mount is still there

Reply:
addressed in ne update